data-serving/samples/cases.json

```diff
@@ -2,16 +2,14 @@
     {
         "demographics": {
             "ageRange": {
-                "start": {
-                    "$numberDouble": "50"
```

Comment:
What's the impact of removing this?

Reply:
it used to be that start/end had to be doubles, but @allysonjp715 changed that last week and now it can be an int or double